S3/input.tf

```diff
@@ -83,3 +83,9 @@ variable "object_lock_configuration" {
   type        = any
   default     = {}
 }
+
+variable "replication_configuration" {
+  description = "(Optional) Map containing cross-region replication configuration."
+  type        = any
+  default     = {}
+}
```

Comment:
Do we need the level of configuration this gives us? 

Do we have a use case for this other than replicate everything in a bucket into another bucket in the same region?

Reply:
Should this just be the ARN of the destination bucket for simplicity's sake?

Reply:
For cloud based sensor, I'll need to control encryption with a KMS key and changing object ownership.

Things like `metrics` and `replication_time` could be dropped with no problem.  I could also wipe out the `filters` and just use full object replication (default to a blank filter).

Reply:
Yes I'd like to keep it as simple as possible, we can always add in the complexity if needed later.